### PR TITLE
Pretty Printing done in loadSourceText .

### LINKED ISSUE #8071: loadSourceText should probably be responsible for pretty-printing files
Right now pretty-printing is its own action that kind of hacks its way into the source-loading process. This isn't idea because it means that the promise returned by `loadSourceText` can't consistently handle the case where a file is in the process of being pretty-printed, since don't have a way to know when pretty-printing completes.

### PR DESCRIPTION
**Pull request by nagpalm7**
Fixes #8071 

### Summary of Changes

* Create prettyPrintSources function which returns source with and sets desirable mappings.

### CODE REVIEW

File: src/actions/sources/prettyPrint.js [hunk 1]
```diff
@@ -50,18 +74,11 @@ export function createPrettySource(sourceId: string) {
...
-    for (const sourceActor of source.actors) {
-      await sourceMaps.applySourceMap(sourceActor.actor, url, code, mappings);
-    }
+    const { text } = await prettyPrintSource(sourceMaps, prettySource, source);
 
     const loadedPrettySource: JsSource = {
       ...prettySource,
-      text: code,
+      text: text,
       loadedState: "loaded"
```
Comment (loganfsmyth) [inline src/actions/sources/prettyPrint.js]: One of the motivations behind the issue for this is that `loadSourceText` already does this `loaded` toggling as part of the `LOAD_SOURCE_TEXT` action for normal sources. I'm hoping that by doing this in loadSourceText that we can remove this part from here entirely.

Comment (nagpalm7) [reply src/actions/sources/prettyPrint.js]: Okay @loganfsmyth, I want to know whether we want to prettyprint all files on their initial loading and remove the dependence on clicking toggle button?

File: src/actions/tests/__snapshots__/project-text-search.spec.js.snap [hunk 2]
```diff
@@ -82,26 +82,7 @@ Array [
...
-        "type": "MATCH",
-        "value": "function bla(x, y) {",
-      },
-    ],
-    "sourceId": "bar:formatted",
-    "type": "RESULT",
-  },
-]
-`;
+exports[`project text search should ignore sources with minified versions 1`] = `Array []`;
```
Comment (jasonLaster) [inline src/actions/tests/__snapshots__/project-text-search.spec.js.snap]: hmm, i wonder why this changed?

Comment (nagpalm7) [reply src/actions/tests/__snapshots__/project-text-search.spec.js.snap]: @jasonLaster No idea, `yarn jest -u` did this work.

Comment (jasonLaster) [reply src/actions/tests/__snapshots__/project-text-search.spec.js.snap]: I think it is worth looking into whether this is desired :)

**Comment by jasonLaster** [comment]
Thanks @nagpalm7! 

This will be landing pretty soon - https://phabricator.services.mozilla.com/D23917?vs=77402&id=77949#toc

**Pull request closed without merging**